openml/runs/functions.py

```diff
@@ -87,20 +126,25 @@ def _run_task_get_arffcontent(model, task, class_labels):
             testX = X[test_indices]
             testY = Y[test_indices]
 
-            model.fit(trainX, trainY)
-            if isinstance(model, BaseSearchCV):
-                _add_results_to_arfftrace(arff_tracecontent, fold_no, model,
-                                          rep_no)
+            try:
+                model.fit(trainX, trainY)
+
+                if isinstance(model, BaseSearchCV):
+                    _add_results_to_arfftrace(arff_tracecontent, fold_no, model, rep_no)
+                    model_classes = model.best_estimator_.classes_
+                else:
+                    model_classes = model.classes_
+            except AttributeError as e:
```

Comment:
It all depends on whether we want openml-python to upload runs with client errors or block them all. 
In the prior case, we should catch. In the other case, we should not.